.github/workflows/wheels-pure-build.yml

```diff
@@ -94,9 +106,11 @@ jobs:
 
         if [ "${{ inputs.uses-versioneer }}" = true ]; then
           export RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE="${versioneer_override}"
-        else
-          bash ci/release/apply_wheel_modifications.sh ${versioneer_override}
-          echo "The git diff after modifying the versions is:"
+        fi
+
+        if [ "${{ inputs.uses-versioneer }}" = false ] || [ "${{ inputs.uses-dynamic-name }}" = false ]; then
```

Comment:
Just checking; is equality in bash a single equals sign? Or is that for single brackets?

Reply:
Lol this is one of those things I hate, a lot of these things depend on which shell and what you have. I believe single equals is equality and is standard (i.e. works across different shell types). For double equals the behavior is shell-dependent, but in bash it would give you the same result for booleans. And yes, this is in single brackets; double brackets are also an extension of `sh` but they work on all commonly used shells today.

We could rewrite this statement in a lot of different equivalent ways, I just picked one that worked :shrug: